File: competidores/A03/maior.py
N1, N2, N3, N4 = 0,0,0,0

OP1 = ""
OP2 = ""


N1 = int (raw_input())
OP1 = (raw_input())
N2 = int (raw_input())
N3 = int (raw_input())
OP2 = (raw_input())
N4 = int (raw_input())

if (OP1 == "+" and OP2 == "+"):
    if(N1 + N2) > (N3 + N4):
        print ">"
    else:
        if (N1 + N2) < (N3 + N4):
            print "<"
        else:
            print "="
elif (OP1 == '+' and OP2 == '*'):
    if(N1 + N2) > (N3 * N4):
        print ">"
    else:
        if (N1 + N2) < (N3 * N4):
            print "<"
        else:
            print "="  
elif (OP1 == '+' and OP2 == '^'):
    if(N1 + N2) > (N3 ** N4):
        print ">"
    else:
        if (N1 + N2) < (N3 ** N4):
            print "<"
        else:
            print "="  
elif (OP1 == '*' and OP2 == '+'):
    if(N1 * N2) > (N3 + N4):
        print ">"
    else:
        if (N1 * N2) < (N3 + N4):
            print "<"
        else:
            print "=" 
elif (OP1 == '*' and OP2 == '*'):
    if(N1 * N2) > (N3 * N4):
        print ">"
    else:
        if (N1 * N2) < (N3 * N4):
            print "<"
        else:
            print "=" 
elif (OP1 == '*' and OP2 == '^'):
    if(N1 * N2) > (N3 ** N4):
        print ">"
    else:
        if (N1 * N2) < (N3 ** N4):
            print "<"
        else:
            print "="
elif (OP1 == '^' and OP2 == '+'):
    if (N1 ** N2) > (N3 + N4):
        print ">"
    else:
        if (N1 ** N2) < (N3 + N4):
            print "<"
        else:
            print"="
elif (OP1 == '^' and OP2 == '*'):
    if(N1 ** N2) > (N3 * N4):
        print ">"
    else:
        if (N1 ** N2) < (N3 * N4):
            print "<"
        else:
            print "="
elif (OP1 == '^' and OP2 == '^'):
    if(N1 ** N2) > (N3 ** N4):
        print ">"
    else:
        if (N1 ** N2) < (N3 ** N4):
            print "<"
        else:
            print "="

File: competidores/P55/maior.java
import java.util.Scanner;
public class maior {

	public static void main(String[] args) {
		
		Scanner sc = new Scanner(System.in);
		int n1 = sc.nextInt();
		String op1 = sc.next();
		int n2 = sc.nextInt();
		int n3 = sc.nextInt();
		String op2 = sc.next();
		int n4 = sc.nextInt();
		
		int result1 = 0;
		int result2 = 0;
		
		if(op1.equals("x")){
			result1 = n1*n2;
		}else if(op1.equals("^")){
			result1 = (int) Math.pow(n1, n2);
		}else if(op1.equals("+")){
			result1 = n1+n2;
		}
		
		if(op2.equals("x")){
			result2 = n3*n4;
		}else if(op2.equals("^")){
			result2 = (int) Math.pow(n3, n4);
		}else if(op2.equals("+")){
			result2 = n3+n4;
		}
		
		if (result1>result2){
			System.out.println(">");
		}else if (result1<result2){
			System.out.println("<");
		}else if (result1==result2){
			System.out.println("=");
		}
	}
}